test/testutil/tests.c

```diff
@@ -19,6 +19,9 @@
 #define MEM_BUFFER_SIZE     (33)
 #define MAX_STRING_WIDTH    (80)
 
+/* Special representation of -0 */
+static char *const BN_minus_zero = "-0";
```

Comment:
So the pointer doesn't get changed.

Declaring it as an array is neater.  I'll update.